### ISSUE TITLE
Landscape Mode - Not Scrollable / Overflow

### ISSUE THREAD

**Issue by ektemfg** [ektemfg/GoogleTranslat0r#1]
To reproduce the issue:

Rotate your device to landscape mode and open the LiveTranslation / Translation History page. Observe that the page is overflowing and the user is unable to scroll.

Expected result: The app should be shown horizontally on the screen and the user should be able to scroll up and down to view their LiveTranslation / Translation History without any overflows or strange appearances.

Actual result: When the device is rotated to landscape mode and the LiveTranslation / Translation History page is opened, the user is unable to scroll and the bottom of the page is overflowing by 84 pixels.

![image](https://user-images.githubusercontent.com/99316570/204928938-adb9a0bc-7f2e-4729-b904-3b849cb271ce.png)

Potential fixes:
ScrollView? Scrollable?
Will update the issue with actual fix when fixed.

**Comment by ksloginov** [comment]
Another potential fix/suggestion: prevent landscape orientation for the app. (Though, it's not an ideal fix)

**Comment by ektemfg** [comment]
Dear @ksloginov 
Thank you for feedback and suggestions!
We are now one step closer to fixing this issue.
Dimitrije

**Comment by ektemfg** [comment]
Hello again @ksloginov,
We apologise for the delay. This issue has been a hard nut to crack. We decided to prevent app being used in landscape mode, but after our users gave us feedback that they really want it back, we were able to find a better solution.
The landscape mode bug that you reported has been fixed, and the app is now usable in both landscape and portrait modes. 
<img width="850" alt="landscape1" src="https://user-images.githubusercontent.com/99316570/207757105-aa446423-13f4-4d75-9c0b-74b8d28a31c7.png">
<img width="850" alt="landscape2" src="https://user-images.githubusercontent.com/99316570/207757084-bdba00b1-2c42-43d1-9b5e-f75d0b0e587c.png">
<img width="850" alt="landscape" src="https://user-images.githubusercontent.com/99316570/207757375-a79cd518-cb9a-474a-8433-1f28a0ad513a.png">
The changes will be pushed to the repository after they have been reviewed and tested by our team.
Thank you!
Dimitrije
